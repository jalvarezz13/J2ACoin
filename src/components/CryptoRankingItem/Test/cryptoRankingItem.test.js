import React from 'react'
import renderer from 'react-test-renderer'
import Link from '../Link'

it('renders correctly', () => {
  const tree = renderer
    .create(<Link page="https://coinmarketcap.com/">CoinMarketCap</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
