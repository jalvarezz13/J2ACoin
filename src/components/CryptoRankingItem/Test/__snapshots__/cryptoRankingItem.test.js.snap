// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="item-container"
>
  <p
    className="item-id"
  >
    1
     
  </p>
  <img
    alt="Logo de Bitcoin"
    className="item-logo"
    src="https://s2.coinmarketcap.com/static/img/coins/64x64/1.png"
  />
  <p
    className="item-name"
  >
    Bitcoin
     (
    BTC
    )
  </p>
  <div
    className="price-container"
  >
    <span
      className="item-price"
    >
      44379.3465
       $
    </span>
    <img
      alt="PoW"
      className="proof-of"
      src="pick.png"
    />
  </div>
  <p
    className="change-value-negative"
  >
    -0.0459
    %
  </p>
  <p>
    842.968.732.920,82
    $
  </p>
  <div
    className="item-change-7d"
  >
    <div
      className="item-change-7d-text"
    >
      <p>
        7 días
      </p>
      <p
        className="change-value-positive"
      >
        5.8318
        %
      </p>
    </div>
    <img
      alt="Gráfico a 7 días de Bitcoin"
      className="graph-7d-positive"
      src="https://s3.coinmarketcap.com/generated/sparklines/web/7d/2781/1.svg"
    />
  </div>
</div>
`;
